Dangerfile

```diff
@@ -40,7 +40,7 @@
 # content changes within the diff
 @S_SECURITY_CONTENT_PATTERN ||= /auth|password|permission|token|secret|security|scope|api_key|apikey|sudo/
 # dont ever match against changes in these files
-@S_SECURITY_EXCLUDE_FILES ||= /test_.*\.py/
+@S_SECURITY_EXCLUDE_FILES ||= /CHANGES|tests/
```

Comment:
Seemed more reliable to just capture `tests` instead of `test_*.py` since we have many files named like, `tests.py`, and all of the js tests, etc.